be/src/exec/pipeline/scan/connector_scan_operator.cpp

```diff
@@ -57,15 +57,13 @@ void ConnectorScanOperator::attach_chunk_source(int32_t source_index) {
     auto* factory = down_cast<ConnectorScanOperatorFactory*>(_factory);
     auto& active_inputs = factory->get_active_inputs();
     auto key = std::make_pair(_driver_sequence, source_index);
-    DCHECK(!active_inputs.contains(key));
```

Comment:
why delete the `DCHECK`?

Reply:
Because we detach chunk source early, so may violate these constraints.